src/misc.jl

```diff
@@ -32,9 +32,9 @@ function concretize(xss::AbstractVector...)
         @label next_j1
     end
 
-    yss = Array{AbstractVector}(length(xss))
+    yss = Vector{AbstractVector}(undef, length(xss))
     for (i, xs) in enumerate(xss)
-        yss[i] = Array{eltype(xs)}(count)
+        yss[i] = Vector{eltype(xs)}(undef, count)
```

Comment:
wow, how did you find this omission?  would be great to add a unit test, as codevec is reporting that this codepath is still not tested.

Reply:
Ouch that must've slipped by us when we updated everything for 1.0. We need a unit test there ASAP

Reply:
I'll add a unit test to this PR. I found this omission while investigating #1167, after restyling `Geom.ribbon`.